repose-aggregator/components/services/akka-http-client-service/akka-http-client-service-impl/src/main/java/org/openrepose/core/services/serviceclient/akka/impl/AkkaServiceClientImpl.java

```diff
@@ -103,10 +113,39 @@ public ServiceClientResponse get(String hashKey, String uri, Map<String, String>
 
     @Override
     public ServiceClientResponse get(String hashKey, String uri, Map<String, String> headers, boolean checkCache) throws AkkaServiceClientException {
-        AuthGetRequest authGetRequest = new AuthGetRequest(hashKey, uri, headers);
+
+
         try {
             Timeout timeout = new Timeout(socketTimeout + CONNECTION_TIMEOUT_BUFFER_MILLIS, TimeUnit.MILLISECONDS);
-            Future<ServiceClientResponse> future = getFuture(authGetRequest, timeout, checkCache);
+            Future<ServiceClientResponse> future;
+
+            if (openTracingService.isEnabled()) {
```

Comment:
Is the open tracing api not written in a way where it's safe for us always to call it regardless of how the back end is configured? Kind of log4j/slf4j where we can always safely make our calls to loggers regardless of what level or output, if any, they are configured for.